applications/NXstress.nxdl.xml

```diff
@@ -0,0 +1,1115 @@
+                    <doc>File name(s) and/or path(s) (withing file(s)) containing data from the last calibration(s). This can be a list.</doc>
+                </field>
+                <field name="data" type="NX_BINARY" minOccurs="0" maxOccurs="1">
+                    <doc>Calibration file content.</doc>
+                </field>
+                <field name="type" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>Mime content type of calibration *data* field e.g. text/plain, application/json,...</doc>
+                </field>
+                <field name="author" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>Author or creator of the calibration.</doc>
+                </field>
+                <field name="date" type="NX_DATE_TIME" minOccurs="0" maxOccurs="1">
+                    <doc>Date calibration was created/added</doc>
+                </field>
+            </group>
+            <group name="SOURCE" nameType="any" type="NXsource" minOccurs="1" maxOccurs="1">
+                <field name="type" type="NX_CHAR" minOccurs="1" maxOccurs="1">
+                    <doc>
+                        Type of radiation source
+                    </doc>
+                    <enumeration>
+                        <item value="Spallation Neutron Source"/>
+                        <item value="Pulsed Reactor Neutron Source"/>
+                        <item value="Reactor Neutron Source"/>
+                        <item value="Synchrotron X-ray Source"/>
+                        <item value="Rotating Anode X-ray"/>
+                        <item value="Fixed Tube X-ray"/>
+                        <item value="Metal Jet X-ray"/>
+                    </enumeration>
+                </field>
+                <field name="probe" type="NX_CHAR" minOccurs="1" maxOccurs="1">
+                    <doc>
+                        Type of radiation probe
+                    </doc>
+                    <enumeration>
+                        <item value="neutron"/>
+                        <item value="X-ray"/>
+                    </enumeration>
+                </field>
+            </group>
+            <group name="DETECTOR" nameType="any" type="NXdetector" minOccurs="1" maxOccurs="unbounded">
+                <doc>Zero or more of these groups describe the detectors used in the experiment.</doc>
+                <field name="description" type="NX_CHAR" minOccurs="0" maxOccurs="1">
+                    <doc>name/manufacturer/model/etc. information</doc>
+                </field>
+                <field name="type" type="NX_CHAR">
+		            <doc>
+			                Description of type such as \ :sup:`3`\ He gas cylinder, \ :sup:`3`\ He PSD, scintillator, fission chamber, proportion counter, ion chamber, CCD, pixel, image plate, CMOS, …
+		            </doc>
+	            </field>
+                <field name="distance" type="NX_NUMBER" units="NX_LENGTH" minOccurs="0" maxOccurs="1">

```

Comment:
This changes the meaning of the field from the base class

Reply:
The `dimensions` was missing indeed. I added them + the symbols.